managers/elasticsearch.py

```diff
@@ -29,14 +29,32 @@ def createElasticConnection(self, scheme=None, host=None, port=None, user=None,
 
         creds = '{}:{}@'.format(user, pswd) if user and pswd else ''
 
-        host = '{}://{}{}:{}'.format(scheme, creds, host, port)
+        multHosts = []
+
+        if isinstance(host, list) or \
+            isinstance(host, dict) or \
+            isinstance(host, set) or \
+            isinstance(host, tuple):
+            
+            for i in host:
```

Comment:
I think it might be necessary for `host` to be a string because of how it's stored in AWS, not sure we can get it out of there as an iterable. Would just need to pick a delimiter and then split/iterate on that.

Reply:
That makes sense so I'll pick a comma with a space as the delimiter and split then iterate on that.